package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "3.4.0",
+  "version": "4.0.0",
```

Comment:
**note:** Bump to v4 ready for tagging, to test out the updated autofix action. Major bump due to the Node 20 stuff.